src/test/java/seedu/workbook/storage/JsonAdaptedInternshipTest.java

```diff
@@ -41,75 +44,89 @@ public void toModelType_validInternshipDetails_returnsInternship() throws Except
     @Test
     public void toModelType_invalidCompany_throwsIllegalValueException() {
         JsonAdaptedInternship internship = new JsonAdaptedInternship(INVALID_COMPANY, VALID_ROLE, VALID_PHONE,
-                VALID_EMAIL, VALID_TAGS);
+                VALID_EMAIL, VALID_STAGE, VALID_TAGS);
         String expectedMessage = Company.MESSAGE_CONSTRAINTS;
         assertThrows(IllegalValueException.class, expectedMessage, internship::toModelType);
     }
 
     @Test
     public void toModelType_nullCompany_throwsIllegalValueException() {
-        JsonAdaptedInternship internship = new JsonAdaptedInternship(null, VALID_ROLE, VALID_PHONE, VALID_EMAIL,
-                 VALID_TAGS);
+        JsonAdaptedInternship internship = new JsonAdaptedInternship(null, VALID_ROLE, VALID_PHONE,
+                VALID_EMAIL, VALID_STAGE, VALID_TAGS);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Company.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, internship::toModelType);
     }
 
     @Test
     public void toModelType_invalidRole_throwsIllegalValueException() {
         JsonAdaptedInternship internship = new JsonAdaptedInternship(VALID_COMPANY, INVALID_ROLE, VALID_PHONE,
-                VALID_EMAIL, VALID_TAGS);
+                VALID_STAGE, VALID_EMAIL, VALID_TAGS);
```

Comment:
order of arguments should be `VALID_EMAIL` then `VALID_STAGE`? based on the `JsonAdaptedInternship` constructor. the test case will still pass due to the way it is set up but would be good to fix :)

Reply:
Resolved. Thank you for picking that up!